apps/shell/app.js

```diff
@@ -55,9 +57,44 @@ app.use(process.env.PASSENGER_BASE_URI || '/', router);
 var server = new http.createServer(app);
 var wss = new WebSocket.Server({ noServer: true });
```

Comment:
Make these constants.

```suggestion
const server = new http.createServer(app);
const wss = new WebSocket.Server({ noServer: true });
```